src/commands/publish/upload_form/plain_text.rs

```diff
@@ -0,0 +1,18 @@
+use super::binding::Binding;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Deserialize, Serialize)]
+pub struct PlainText {
+    pub name: String,
+    pub value: String,
```

Comment:
thought we called this `text` not value? @ashleymichal

Reply:
this is where the binding is built https://github.com/cloudflare/wrangler/pull/1014/files#diff-1faa2c3f00938a7470735a3979b5ddbeR26